pkg/security/oauth/manager/manager.go

```diff
@@ -2,17 +2,15 @@ package manager
 
```

Comment:
OAuth manager is still needed?

Reply:
I think it can be removed, but I want to fix re-enable and fix all tests first before I start removing code.